ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/WorkloadCves.helpers.js

```diff
@@ -236,7 +236,8 @@ export function visitAnyImageSinglePage() {
     cy.get('tbody tr td[data-label="Image"] a').first().click();
 
     return cy.get('h1').then(($h1) => {
-        return $h1.text().split(':');
+        // Remove the SHA and/or tag from the image name
+        return $h1.text().replace(/(@sha256)?:.*/, '');
```

Comment:
The new ordering of the tables revealed a possible issue in the tests. If an image with a SHA instead of tag is selected, the assertions break down. This now handles both SHA and tag when parsing out the image name.